easybuild/easyconfigs/d/dadi/dadi-1.7.0-intel-2016b-Python-2.7.12.eb

```diff
@@ -23,4 +23,8 @@ sanity_check_paths = {
     'dirs': ['lib/python%(pyshortver)s/site-packages'],
 }
 
+sanity_check_commands = [
+    ('cd %(builddir)s/gutenkunstlab-dadi-01068e3ad50d/tests && python run_tests.py', '')
```

Comment:
@tjslauson the `gutenkunstlab-dadi-01068e3ad50d` subdirectory looks very version-specific, which means we'll need to update it on every version bump (i.e. this won't play nicely with `--try-software-version`)

So, we should try to make it generic, e.g. using `cd %(builddir)s/*/tests && python run_tests.py`.

You can also include just strings in `sanity_check_commands` now btw rather than tuples (which was weird syntax anyway), so I'd suggest to use:

```python
sanity_check_commands = ["cd %(builddir)s/*/tests && python run_tests.py"]
```